WORKSPACE

```diff
@@ -1,3 +1 @@
-workspace(name = "player")
-load("//:shell.bzl", "sh_configure")
-sh_configure()
```

Comment:
What does this change do?

Reply:
We are already doing this in `MODULE.bazel` [here](https://github.com/player-ui/player/blob/main/MODULE.bazel#L31-L34). This was from before when the shell configuration was done locally and not via our rules.